streams/src/test/java/org/apache/kafka/streams/state/internals/AbstractSessionBytesStoreTest.java

```diff
@@ -121,15 +121,15 @@ public void shouldPutAndFindSessionsInRange() {
 
         try (final KeyValueIterator<Windowed<String>, Long> values = sessionStore.findSessions(key, 0, 1000L)
         ) {
-            assertEquals(new HashSet<>(expected), toSet(values));
+            assertEquals(expected, toList(values));
```

Comment:
before this PR, we only verify the returned data `contains` in the expected data as **Set**. Now, we'll change to **List**, which means, not only the data should be correct, but also the data order should be correct.